src/Microsoft.Azure.WebJobs.Host/Listeners/ListenerFactoryContext.cs

```diff
@@ -39,6 +43,11 @@ public ListenerFactoryContext(FunctionDescriptor descriptor, ITriggeredFunctionE
         public ITriggeredFunctionExecutor Executor { get; private set; }
 
         /// <summary>
+        /// Gets the web jobs exception handler
+        /// </summary>
+        public IWebJobsExceptionHandler ExceptionHandler { get; private set; }
```

Comment:
The configs create the listeners. So could they just cache this and pass to the listeners?  Put another way, is adding this here enabling something new, or just encouraging an existing pattern?

---
In reply to: [109277405](https://github.com/Azure/azure-webjobs-sdk/pull/1100#discussion_r109277405) [](ancestors = 109277405)